javascript/cable_ready.js

```diff
@@ -158,7 +158,7 @@ const DOMOperations = {
   }
 }
 
-const perform = operations => {
+const perform = (operations, options = { processMissing: true }) => {
```

Comment:
I like the idea of the options hash, but would prefer something like this for the name and default value. 

```js
{ignoreMissingElements: false}
```